README.md

```diff
@@ -27,6 +27,9 @@
 -->
 
 [![License](https://img.shields.io/badge/License-BSD3-lightgrey.svg)](https://opensource.org/licenses/BSD-3-Clause)
+![Static Badge](https://img.shields.io/badge/Triton-24.08-8A2BE2)
+![Static Badge](https://img.shields.io/badge/vLLM-0.5.5-blue)
+![Static Badge](https://img.shields.io/badge/CI_Passing-V100%2CA100%2CH100-Green)
```

Comment:
could you please clarify how these static badges work?

Reply:
Ideally this should be automated. But, it is a manual process at the moment. Once the cron task is finished and the pipeline is green, I'd have to issue a PR to update the badges here.

Reply:
Do we want to hold on adding badges, until we have an automated workflow in place?